third-party/realdds/src/dds-notification-server.cpp

```diff
@@ -84,8 +84,11 @@ dds_notification_server::dds_notification_server( std::shared_ptr< dds_publisher
     } );
 
     dds_topic_writer::qos wqos( RELIABLE_RELIABILITY_QOS );
-    wqos.history().depth = 10;                                                                      // default is 1
-    wqos.endpoint().history_memory_policy = eprosima::fastrtps::rtps::DYNAMIC_RESERVE_MEMORY_MODE;  // TODO: why?
+    wqos.history().depth = 10; // default is 1
```

Comment:
Maybe we should make this our default, to match the reader?

Reply:
I don't think we need 10 as default, it would be wasteful, especially for frames.
Here is probably a copy-paste from reader leftover, I'll remove.

Note, to actually receive all previously sent samples, a new reader must be `reliable` and durability, on reader and writer, must be `transient_local`

Reply:
I agree that for frames should not have a history.
Depth=1
Frames history is implemented at LibRS level.
Can communication should only keep 1 and not an history.